### ISSUE TITLE
Buffering enabled on k8s service throws errors about /tmp not available in non-alpine docker image

### ISSUE THREAD

**Issue by oivindoh** [containous/traefik#4064]
### Do you want to request a *feature* or report a *bug*?

Bug

### What did you do?

Enabled buffering via traefik.ingress.kubernetes.io/buffering annotation on kubernetes services

```yaml
    traefik.ingress.kubernetes.io/buffering: |
      maxrequestbodybytes: 10485760
      memrequestbodybytes: 2097153
      maxresponsebodybytes: 10485761
      memresponsebodybytes: 2097152
      retryExpression: IsNetworkError() && Attempts() <= 2
```

Running traefik:1.7.3 due to http handler panic in 1.7.2 with same configuration.

### What did you expect to see?

Browse to site (GET), get served the content normally available on the URI as long as request body is not too large.

### What did you see instead?

No content, just a HTTP 200 instead of expected content, evne with no request body on a GET request.

In traefik's log, this message is spammed en masse:
```
time="2018-10-17T07:51:09Z" level=error msg="open /tmp/temp-multibuf-691319615: no such file or directory"
```
It would appear /tmp is not writable in the non-alpine container. Switching to 1.7.3-alpine allows the configuration to work, and content is received as expected.

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
(paste your output here)
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

Same environment/configuration as in #4005, but with 1.7.3 versions of docker hub image.

### If applicable, please paste the log output in DEBUG level (`--logLevel=DEBUG` switch)

```
(paste your output here)
```

**Comment by dtomcej** [comment]
@dduportal The current scratch image doesn't have a `/tmp` dir, but is the default dir for the buffering package. Can we add a `/tmp` dir to the scratch dockerfile? or are we aiming to have read-only FS, and this is something we need to discuss further?

**Comment by traefiker** [comment]
Closed by #4083.
